scene/gui/text_edit.cpp

```diff
@@ -212,12 +212,15 @@ float TextEdit::Text::get_indent_offset(int p_line, bool p_rtl) const {
 	return text_line.indent_ofs;
 }
 
-_FORCE_INLINE_ String TextEdit::Text::operator[](int p_line) const {
-	ERR_FAIL_INDEX_V(p_line, text.size(), "");
+_FORCE_INLINE_ const String &TextEdit::Text::operator[](int p_line) const {
+	static const String empty;
+	ERR_FAIL_INDEX_V(p_line, text.size(), empty);
 	return text[p_line].data;
 }
 
 _FORCE_INLINE_ const String &TextEdit::Text::get_text_with_ime(int p_line) const {
+	static const String empty;
+	ERR_FAIL_INDEX_V(p_line, text.size(), empty);
```

Comment:
I don't think this is needed, this function is only available to TextEdit and it is only called in one place and it has its own bounds check.

Reply:
Alright, I guess I was a little overzealous with this change. I removed both checks; it will as before crash on invalid access now.